samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -37,7 +37,12 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
 
   // Whenever the sample is changed from desktop -> mobile using the emulator, make sure we update the formFactor.
   useEffect(() => {
-    const updateIsMobile = (): void => setIsMobileSession(detectMobileSession());
+    const updateIsMobile = (): void => {
+      // The userAgent string is sometimes not updated synchronously when the `resize` event fires.
+      setImmediate(() => {
```

Comment:
Was this being tested on IE? :D
https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate
![image](https://user-images.githubusercontent.com/2684369/154168574-10fc0725-f4ff-4407-b57c-0973df76e2ee.png)

can do `setTimeout(() => {}, 0)` to get the same effect

Reply:
I think it is ok to not support IE. The underlying calling sdk as an example does not.

Reply:
no no, this is _only_ supported on IE (and edge to some extent)

Reply:
ah that makes more sense .. yes I am also confused why we need to do this.

Reply:
Oooh.... I looked at that page. And also interpreted it as not supported on IE :D